backend/EmailUtils.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+
+EMAIL_TEMPLATE_PATH = "emailTemplate.html"
+
+HTML_TABLE_STR = "<tr> <td>{number}</td> <td>{type}</td> <td>{description}</td> <td>{link}</td> </tr>"
+
+def generateEmailHtmlStr():
+    templateStr = open(EMAIL_TEMPLATE_PATH).read()
+    tableRowStr = ""
+    for i in range(9):#result:
+        tableRowStr += HTML_TABLE_STR.format(number = i, type = "Jobs",
+                                                description = "Need Software Engineer", link = "Link")
+    finalHtmlStr = templateStr.format(tableRowStr)
+    print(finalHtmlStr)
+    return finalHtmlStr
+
+generateEmailHtmlStr()
+
+'''
+Send email on CentOS using the "mail" command.
+Example:
+echo "<html> Hello </html>" | mail -s "$(echo -e "Release Status [Green]\nContent-Type: text/html")" rdalwadi@uci.edu
```

Comment:
Move the email address to the .env file too, so everyone can test it using their email locally.

Reply:
This is just the sample command.  The email with be based on the one provided when calling the /subscribe api.

Reply:
Ah okay, got it!